nerfstudio/scripts/downloads/download_data.py

```diff
@@ -453,6 +456,96 @@ def download(self, save_dir: Path):
         os.remove(download_path)
 
 
+mill19_downloads = {
+    "building": "https://storage.cmusatyalab.org/mega-nerf-data/building-pixsfm.tgz",
+    "rubble": "https://storage.cmusatyalab.org/mega-nerf-data/rubble-pixsfm.tgz",
+    "all": None,
+}
+
+if TYPE_CHECKING:
+    Mill19CaptureName = str
+else:
+    Mill19CaptureName = tyro.extras.literal_type_from_choices(mill19_downloads.keys())
+
+
+@dataclass
+class Mill19Download(DatasetDownload):
+    """Download the Mill 19 dataset."""
+
+    capture_name: Mill19CaptureName = "building"
+
+    def download(self, save_dir: Path):
+        """Download a Mill 19 dataset: https://meganerf.cmusatyalab.org/#data"""
+
+        install_checks.check_curl_installed()
+        if self.capture_name == "all":
+            for capture_name in mill19_downloads:
+                if capture_name != "all":
+                    Mill19Download(capture_name=capture_name).download(save_dir)
+            return
+
+        assert (
+            self.capture_name in mill19_downloads
+        ), f"Capture name {self.capture_name} not found in {mill19_downloads.keys()}"
+        url = mill19_downloads[self.capture_name]
+        target_path = str(save_dir / f"mill19/{self.capture_name}")
+        os.makedirs(target_path, exist_ok=True)
```

Comment:
can we use the `pathlib` API for these operations instead? like:
```
target_path = save_dir / ...
target_path.mkdir(exist_ok=True)
```

same for replacing `os.listdir()`, `os.path.join()`, etc below